modules/nf-core/nanofilt/main.nf

```diff
@@ -0,0 +1,51 @@
+process NANOFILT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanofilt:2.8.0--py_0':
+        'biocontainers/nanofilt:2.8.0--py_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val   readlength
+    val   readqual
+
+    output:
+    tuple val(meta), path("*.fastq.gz"), emit: filtreads
+    path "versions.yml"                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "filtered_${meta.id}"
+    """
+    gunzip \\
+        -c $reads \\
+        | NanoFilt \\
+        --maxlength $readlength \\
+        -q $readqual \\
```

Comment:
If there exists a long form of an option (e.g. `--quality`), please use that over the short form (`-q`)

Reply:
With the new changes the `--quality` paramater is no longer needed in those lines, but really nice to know that practice, thank you!